test/index.test.ts

```diff
@@ -0,0 +1,32 @@
+import 'jest';
+import request from 'supertest';
+
+import app from './server';
+
+// // designed to mock the axios request made to the webapp to post a new query
+// const mockAxios = {
+//     post: jest.fn(() => Promise.resolve({ data: {} })),
+// };
+
+/**
+ *
+ * TESTING SUITE
+ *
+ */
+describe('Logger End to End Test', () => {
+    describe('successful query', () => {
+        test('Test API with middleware runs without errors', (done) => {
+            // merely checks if test API can run the middleware without errors
+            request(app)
+                .get('/')
+                .expect(200)
+                .expect('done')
+                .end((err, res) => {
+                    if (err) {
+                        return err;
+                    }
+                    return done();
+                });
+        });
+    });
+});
```

Comment:
I think it's worthwhile to add tests for when bad data is passed in or a connection fails to make sure errors don't disrupt the middleware chain.

Reply:
will take care of this

Reply:
added